docs/appendix/yaml.rst

```diff
@@ -329,7 +329,13 @@ will be used to plot. if use 'epa+region' then a total of 61 regions can be choo
 **region_list:** multi-box plot only. list of regions in the above 'region_name' variable 
 user want to plot.
 
-**model_name_list:** multi-box plot only. list of observation and model names user choose to set as x-labels
+**model_name_list:** 
+for multi-box plot, list of observation and model names user choose to set as x-labels; 
+for csi plot, list of mode names (only) user choose to set as labels.
```

Comment:
Should it be "model" here instead of mode?

Reply:
yes it should. I will update it

Reply:
just fix it